R/toModelResolution.R

```diff
@@ -2,28 +2,29 @@
 #'
 #' Missing periods are interpolated linearly and extrapolated constantly,
 #' additional periods are removed and all other dimensions are filtered to the
-#' elements dined in the model sets.
+#' elements defined in the model sets.
 #'
 #' @param x data.frame with temporal dimension
 #' @param m gams Container with sets as known dimensions
 #' @param value character, name of value column
+#' @param ignoreDims character, dimensions to be neither filtered nor extrapolated
 #' @returns data.frame with temporal resolution according to model
 #'
 #' @author Robin Hasse
 #'
 #' @importFrom quitte interpolate_missing_periods_
 
-toModelResolution <- function(x, m, value = "value") {
+toModelResolution <- function(x, m, value = "value", ignoreDims = NULL) {
```

Comment:
Im wondering if `ignoreDims` is the best argument name, since the all entries are allowed. Maybe something like `unfilteredDims`? But I'm not sure and can certainly live with this name, too.